usd/src/usd_parser/USDData_TEST.cc

```diff
@@ -43,15 +43,19 @@ TEST(USDData, Constructor)
   auto systemPaths = ignition::common::systemPaths();
   systemPaths->AddFilePaths(sdf::testing::TestFile("usd"));
 
-  // Open a valid USD file
   {
+    // Remove the material folder if there is already one created
+    ignition::common::removeAll(
+      ignition::common::joinPaths(ignition::common::cwd(), "materials"));
```

Comment:
Is this necessary? I imagine that if the folder already exists, this test will simply create files in that folder and skip the folder creation step

Reply:
This was failing in the buildfarm, I decided to remove first the folder to avoid potencial issues

Reply:
I believe I found the issue. It looks like a different test created a `materials` directory and didn't delete it when the test was finished, which caused this test to add numeric suffixes to files that already exist. I made a fix in https://github.com/ignitionrobotics/sdformat/pull/895/commits/c60ebc9ccaf761217aca14d4e1b1973f829de453, I believe CI should come back green now :crossed_fingers:

Reply:
Ci is green! :green_circle: